src/main/java/core/basesyntax/MyLinkedList.java

```diff
@@ -3,45 +3,155 @@
 import java.util.List;
 
 public class MyLinkedList<T> implements MyLinkedListInterface<T> {
+    private Note<T> first;
+    private Note<T> last;
+    private int size;
+    private Note<T> boofNode;
+    private Note<T> newNode;
+
     @Override
     public void add(T value) {
+        if (size == 0) {
```

Comment:
You have method isEmpty() that returns you the same boolean result. You can use it here.